doc/developer/style.md

```diff
@@ -97,13 +97,19 @@ _Portions of this section are copied from the PostgreSQL documentation and subje
 
 For error message style, we strive to adhere to the [guidelines](https://www.postgresql.org/docs/current/error-style-guide.html) in the PostgreSQL manual, and use a format that consists of a **required** _primary_ message followed by **optional** _detail_ and _hint_ messages.
 
-**Primary message**: short, factual, and avoids reference to implementation details. Lowercase the first letter and do not end with any punctuation.
+- **Primary message**: short, factual, and avoids reference to implementation details. Lowercase the first letter and do not end with any punctuation.
 
-**Detail message**: factual, use if needed to keep the primary message short or to mention implementation details. Use complete sentences, each of which starts with a capitalized letter and ends with a period. Separate sentences with two spaces.
+- **Detail message**: factual, use if needed to keep the primary message short or to mention implementation details. Use complete sentences, each of which starts with a capitalized letter and ends with a period. Separate sentences with two spaces.
 
-**Hint message**: suggestions about what to do to fix the problem. This should be actionable, and can provide a shortlink to direct the user to more information. Use complete sentences, each of which starts with a capitalized letter and ends with a period. Separate sentences with two spaces.
+- **Hint message**: suggestions about what to do to fix the problem. This should be actionable, and can provide a shortlink to direct the user to more information. Use complete sentences, each of which starts with a capitalized letter and ends with a period. Separate sentences with two spaces.
 
-To use a shortlink in a Hint, add the shortlink to this [file](https://github.com/MaterializeInc/materialize/blob/3efb2c933e4e6c8e694afb1e794c7d8dae368e7d/ci/deploy/website.sh#L29) in your PR to add/modify the hint mesage. Be mindful that we are committed to the upkeep of the referenced link.
+```
+Primary: could not parse column "foo"
+Detail: Column "foo" in table "bar" contains an unsupported type.
```

Comment:
Nit: in https://github.com/MaterializeInc/materialize/pull/15568 we did `Column "bar.foo" contains an unsupported type.`. We should probably not that on the actual error change too.

Reply:
Didn't see that one. I will follow that example!